app/assets/javascripts/clients.js

```diff
@@ -1,13 +0,0 @@
-$(function () {
```

Comment:
Read about jquery and stop doing this shit!!!
http://learn.jquery.com/using-jquery-core/document-ready/